contrib/windows_installer/DOSBox-Staging-setup.iss

```diff
@@ -0,0 +1,130 @@
+
+[Files]
+Source: "program\*"; DestDir: "{app}"; Flags: ignoreversion recursesubdirs createallsubdirs
+Source: "dosbox.exe"; DestDir: "{app}"; DestName: "dosbox.exe"; Flags: ignoreversion; Tasks: not defaultmsvc
+Source: "dosbox_with_debugger.exe"; DestDir: "{app}"; DestName: "dosbox_with_debugger.exe"; Flags: ignoreversion; Tasks: not defaultmsvc
+Source: "dosbox_msvc.exe"; DestDir: "{app}"; DestName: "dosbox.exe"; Flags: ignoreversion; Tasks: defaultmsvc
+Source: "dosbox_msvc_with_debugger.exe"; DestDir: "{app}"; DestName: "dosbox_with_debugger.exe"; Flags: ignoreversion; Tasks: defaultmsvc
+Source: "dosbox_with_console.bat"; DestDir: "{app}"; DestName: "dosbox_with_console.bat"; Flags: ignoreversion
+
+[Icons]
+Name: "{group}\{#DOSBoxAppName}"; Filename: "{app}\dosbox.exe"; Parameters: "-noconsole"
+Name: "{group}\{#DOSBoxAppName} with console"; Filename: "{app}\dosbox_with_console.bat"
+Name: "{group}\{#DOSBoxAppName} with debugger"; Filename: "{app}\dosbox_with_debugger.exe"
+Name: "{group}\Visit {#DOSBoxAppName} website"; Filename: "https://dosbox-staging.github.io/"
+Name: "{autodesktop}\{#DOSBoxAppName}"; Filename: "{app}\dosbox.exe"; Parameters: "-noconsole"; Tasks: desktopicon
+
+[Registry]
+Root: HKA; Subkey: "Software\dosbox-staging"; Flags: uninsdeletekeyifempty
+Root: HKA; Subkey: "Software\dosbox-staging"; ValueType: string; ValueName: "Path"; ValueData: "{app}"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\dosbox-staging"; ValueType: string; ValueName: "Version"; ValueData: "{#DOSBoxAppVersion}"; Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\Directory\shell\dosbox-staging"; ValueType: string; ValueName: ""; ValueData: "Open with {#DOSBoxAppName}"; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletevalue
+Root: HKA; Subkey: "Software\Classes\Directory\shell\dosbox-staging"; ValueType: string; ValueName: "Icon"; ValueData: """{app}\dosbox_with_console.bat"",0"; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\Directory\shell\dosbox-staging\command"; ValueType: string; ValueName: ""; ValueData: """{app}\dosbox_with_console.bat"" ""%v"""; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\Directory\Background\shell\dosbox-staging"; ValueType: string; ValueName: ""; ValueData: "Open with {#DOSBoxAppName}"; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletevalue
+Root: HKA; Subkey: "Software\Classes\Directory\Background\shell\dosbox-staging"; ValueType: string; ValueName: "Icon"; ValueData: """{app}\dosbox_with_console.bat"",0"; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletekey
+Root: HKA; Subkey: "Software\Classes\Directory\Background\shell\dosbox-staging\command"; ValueType: string; ValueName: ""; ValueData: """{app}\dosbox_with_console.bat"" ""%v"""; Check: WizardIsTaskSelected('contextmenu'); Flags: uninsdeletekey
```

Comment:
there are lots of repeating `dosbox-staging` strings - can you make a `define` for them, and swap it in for all of them too?